go-aws-lambda-zip/go.mod

```diff
@@ -0,0 +1,10 @@
+module go-aws-lambda
+
+go 1.17
+
+require (
+	github.com/Rookout/GoSDK v0.1.10-stub
```

Comment:
Why do you require the stub?

Reply:
This is automatic stuff go does when you import the GoSDK module

Reply:
Not if you have the proxy set up and you "go get" a specific version.
If you leave it like this, it shouldn't work.